multi-node/aws/pkg/config/config.go

```diff
@@ -565,7 +565,7 @@ func (c *Cluster) ValidateExistingVPC(existingVPCCIDR string, existingSubnetCIDR
 		}
 	}
 	_, instanceNet, err := net.ParseCIDR(c.InstanceCIDR)
-	if err != nil {
+	if len(c.Subnets) == 0 && err != nil {
```

Comment:
This arises when `subnets` and `existingVpc` fields are both properly defined, which we do not test.